### PR TITLE
Checked my Portfolio project for Accessibility 

### PR DESCRIPTION
**Pull request by anyars-encarta**
In this project, I checked and updated portions of my portfolio project for accessibility. The report of my assessment is presented below:

- [ ] Page titles - I updated the title of the project to "Anyars' Portfolio". Reference: https://github.com/anyars-encarta/anyars-encarta.github.io/blob/4c34ed3c205f549eb8694e49c56be8e481be9573/index.html#L7
- [ ] Image text alternatives - I updated the img alt text to "Hamburger Menu icon". Reference: https://github.com/anyars-encarta/anyars-encarta.github.io/blob/4c34ed3c205f549eb8694e49c56be8e481be9573/index.html#L15

I also updated the img alt text to "Contact Menu icon". Reference: https://github.com/anyars-encarta/anyars-encarta.github.io/blob/4c34ed3c205f549eb8694e49c56be8e481be9573/index.html#L20

I also updated img alt text to null because they are just decorative: https://github.com/anyars-encarta/anyars-encarta.github.io/blob/4c34ed3c205f549eb8694e49c56be8e481be9573/index.html#L238-L266

- [ ] Text headings - No issues found
- [ ] Color contrast - No issues found
- [ ] Resize - No issues found
- [ ] Interaction - Updated labels in the form and set them to hidden. Reference: https://github.com/anyars-encarta/anyars-encarta.github.io/blob/4c34ed3c205f549eb8694e49c56be8e481be9573/index.html#L288-L298
- [ ] Moving content - Hovering the mouse over .my-logo class pauses the animation.
- [ ] Multimedia - The project has no multimedia content.
- [ ] The basic structure of the page - No issues found

### CODE REVIEW

**Review by AbelG101 — COMMENTED** [review]
# Hi @anyars-encarta 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- Please take note that this is a pair programming activity which means each team member needs to have a substantial contribution for the project to be reviewed. So please reach out to your team member and have them add some commits. ✔

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html
Comment (AbelG101) [inline index.html:1]: https://github.com/anyars-encarta/anyars-encarta.github.io/blob/c6bd805ecb93d28506e97f5f2d0e53adfa88357f/index.html#L306-L309
- [ ] Ouch! this is close but kindly make sure each alt text is the functional equivalent of the image it describes as I noticed you're using the "Linkedin" alt text for social media icons that are not Linkedin this causes confusion to people who rely on screen readers to navigate your webpage so please make sure to update it. ✔
- You can also find the same issue here 👇
https://github.com/anyars-encarta/anyars-encarta.github.io/blob/c6bd805ecb93d28506e97f5f2d0e53adfa88357f/index.html#L38-L42

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @anyars-encarta & @Uzair-Manzoor 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Great report evaluation ✔
- Proper use of permalinks to link corresponding code ✔
- All linter checks are passing ✔
- Descriptive PR title✔

## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Eternalgratis — APPROVED** [review]
# Status: Approved 🟢 
Hi @anyars-encarta @Uzair-Manzoor , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Great implementation of all the requirements in this milestone. 👏
 
image

## Highlights
- No Linter error ✅
- Professional documentation ✅
- Descriptive PR summary ✅

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**